scripts/update

```diff
@@ -0,0 +1,153 @@
+    local _dst="$2"
+
+    echo "downloading $_url to $_dst"
+    curl \
+        --silent \
+        --compressed \
+        --fail \
+        --location \
+        --time-cond "$_dst" \
+        --output "$_dst" \
+        "$_url"
+}
+
+function join_by() {
+    local IFS="$1"
+    shift
+    echo "$*"
+}
+
+clone_all
+
+_imports=()
+while read -r _proto; do
+    if [[ "$_proto" == envoy/* ]]; then
+        _imports+=("M${_proto}=github.com/envoyproxy/go-control-plane/$(dirname "$_proto")")
+    elif [[ "$_proto" == xds/* || "$_proto" == udpa/* ]]; then
+        _imports+=("M${_proto}=github.com/cncf/xds/go/$(dirname "$_proto")")
+    fi
+done <"$_deps_dir/github.com/pomerium/pomerium-console/scripts/proto-dependencies.txt"
+_import_paths=$(join_by , "${_imports[@]}")
+
+_proto_files=(
+    activity_log.proto
+    audit_svc.proto
+    databroker_svc.proto
+    policy.proto
+    routes.proto
+    settings.proto
+    namespaces.proto
+    users.proto
+    key_chain.proto
+    console_config.proto
+    report.proto
+    devices.proto
+    external_data_sources.proto
+    events.proto
+    tsdb.proto
+)
+
+echo "installing python dependencies"
+pip install \
+    'wheel==0.37.1' \
+    'protobuf==3.20.3' \
+    'grpcio==1.49.1' \
+    'grpcio-tools==1.48.2'
+
+(
+    cd "$_root_dir"
+
+    echo "generating dependency protobuf code"
+    python -m grpc_tools.protoc \
+        -I "$_deps_dir/github.com/cncf/xds" \
+        -I "$_deps_dir/github.com/googleapis/googleapis" \
+        -I "$_deps_dir/github.com/envoyproxy/protoc-gen-validate" \
+        -I "$_deps_dir/github.com/envoyproxy/data-plane-api" \
+        -I "$_deps_dir/github.com/census-instrumentation/opencensus-proto/src" \
+        -I "$_deps_dir/github.com/open-telemetry/opentelemetry-proto" \
+        -I "$_deps_dir/github.com/prometheus/client_model" \
+        --python_out="src" \
+        $(cd deps/github.com/envoyproxy/data-plane-api && find . -regex '.*\.proto' | sed 's|^./||') \
+        $(cd deps/github.com/cncf/xds && find . -regex '.*\.proto' | sed 's|^./||') \
+        $(cd deps/github.com/envoyproxy/protoc-gen-validate && find . -regex '.*\.proto' | sed 's|^./||')
+
+    echo "generating pomerium-console protobuf code"
+    mkdir -p src/pomerium/pb
+    python -m grpc_tools.protoc \
+        -I "$_deps_dir" \
+        -I "$_deps_dir/github.com/cncf/xds" \
+        -I "$_deps_dir/github.com/envoyproxy/data-plane-api" \
+        -I "$_deps_dir/github.com/envoyproxy/protoc-gen-validate" \
+        -I "$_deps_dir/github.com/googleapis/googleapis" \
+        -I "$_deps_dir/github.com/pomerium/pomerium/pkg/grpc" \
+        -I "$_deps_dir/github.com/pomerium/pomerium-console/pkg/pb" \
+        --python_out="src/pomerium/pb" \
+        --grpc_python_out="src/pomerium/pb" \
+        "${_proto_files[@]}"
+
+    # fix package names for grpc
+    for _file in src/pomerium/pb/*.py; do
+        replace-in-file 's/^import ([a-z_]+)_pb2/import pomerium.pb.\1_pb2/' "$_file"
+    done
```

Comment:
I was trying to find a way to work around not modifying the generated files but apparently it's impossible without reorganizing origin protobuf files.